ruby/ql/lib/codeql/ruby/security/UnsafeDeserializationCustomizations.qll

```diff
@@ -48,13 +48,12 @@ module UnsafeDeserialization {
   }
 
   /**
-   * An argument in a call to `YAML.load`, considered a sink
-   * for unsafe deserialization. The `YAML` module is an alias of `Psych` in
-   * recent versions of Ruby.
+   * An argument in a call to `YAML.load`, considered a sink for unsafe
+   * deserialization.
    */
   class YamlLoadArgument extends Sink {
     YamlLoadArgument() {
-      this = API::getTopLevelMember(["YAML", "Psych"]).getAMethodCall("load").getArgument(0)
+      this = API::getTopLevelMember("YAML").getAMethodCall("load").getArgument(0)
```

Comment:
Why did you remove `Psych`? Those two modules are the same thing in Ruby 3.0

Reply:
No idea why that happened. Maybe it was a bad merge?